src/controllers/mainController.ts

```diff
@@ -86,6 +86,7 @@ export default class MainController implements vscode.Disposable {
         includeEstimatedExecutionPlanXml: false,
         includeActualExecutionPlanXml: false,
     };
+    private _actualPlanStatuses: string[] = [];
```

Comment:
Consider adding comments around _actualPlanStatuses usage in different functions to clarify the state updates, especially in cleanup and toggle functions.